### PR TITLE
Feat/ support inactive state for day

### PR DESCRIPTION
**Pull request by lidord-wix**
Add support for inactive days (in addition to disabled days)

### CODE REVIEW

File: src/calendar/day/marking/index.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ export interface MarkingProps extends DotProps {
   marked?: boolean;
   today?: boolean;
   disabled?: boolean;
+  inactive?: boolean;
```
Comment (Inbal-Tish) [inline src/calendar/day/marking/index.tsx:44]: Where are you passing 'inactive' to the Marking component? I don't see it in BasicDay...

File: example/src/screens/calendars.tsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ const CalendarsScreen = () => {
       <Fragment>
         <Text style={styles.text}>Calendar with selectable date</Text>
         <Calendar
+          disableAllTouchEventsForInactiveDays
```
Comment (Inbal-Tish) [inline example/src/screens/calendars.tsx:27]: Can you please have a separate example for the inactive state?

File: src/calendar/day/marking/index.tsx (hunk 1 above)
Comment (lidord-wix) [reply src/calendar/day/marking/index.tsx:44]: The type of BasicDay's `marking` is `MarkingProps` from the `Marking` component, so I was had to add it for using `this.marking.inactive` in BasicDay

File: example/src/screens/calendars.tsx (hunk 2 above)
Comment (lidord-wix) [reply example/src/screens/calendars.tsx:27]: sure

File: src/calendar/day/marking/index.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply src/calendar/day/marking/index.tsx:44]: That's great, but you're not passing this prop in BasicDay component (here: https://github.com/wix/react-native-calendars/blob/master/src/calendar/day/basic/index.tsx#L170)

Comment (Inbal-Tish) [reply src/calendar/day/marking/index.tsx:44]: You need to pass to the marking in order to style the marking according to the inactive state as you do in the Day component, no?

Comment (lidord-wix) [reply src/calendar/day/marking/index.tsx:44]: I'm not sure, there's another inactive marking besides the inactive text color?

Comment (Inbal-Tish) [reply src/calendar/day/marking/index.tsx:44]: There is no inactive marking, that's what I'm talking about. What will be the case of an inactive day with marking? How would it look?

Comment (lidord-wix) [reply src/calendar/day/marking/index.tsx:44]: I see.. Added a support for that :)

Comment (Inbal-Tish) [reply src/calendar/day/marking/index.tsx:44]: First, there are other marking beside Dot. Second, how will the inactive state get to the Marking and the Dot components if you're not passing them from BasicDay to the Marking?

Comment (lidord-wix) [reply src/calendar/day/marking/index.tsx:44]: fixed

File: src/calendar/day/basic/index.tsx [hunk 3]
```diff
@@ -174,6 +187,7 @@ export default class BasicDay extends Component<BasicDayProps> {
         marked={this.isMultiDot() ? true : marked}
         selected={this.isSelected()}
         disabled={this.isDisabled()}
+        inactive={this.isInactive()}
```
Comment (Inbal-Tish) [inline src/calendar/day/basic/index.tsx:190]: Yes! That's what I meant

File: src/types.ts [hunk 4]
```diff
@@ -12,6 +12,8 @@ export type DateData = {
   timestamp: number,
   dateString: string
 }
+export type DayState = 'selected' | 'disabled' | 'inactive' | 'today' | '';
```
Comment (Inbal-Tish) [inline src/types.ts:15]: Love it! I was about to do it myself...

Comment (Inbal-Tish) [reply src/types.ts:15]: Why the empty string?

**Pull request merged**